doc/articles/faq.md

```diff
@@ -0,0 +1,268 @@
+# Uno.UI Platorms Frequently Asked Questions
+
+## How do I update the to latest Uno.UI nuget package, I don't see any update ?
+
+You may want to try our latest dev bits, and here's how to do it:
+
+- On your solution node, select **Manage Nuget packages for Solution**
+- Next to the Search text box, check the **Include prerelease** box
+- You should now see the latest Uno.UI package in the **Updates** tab.
+
+## C# Edit and Continue does not work
+
+There's an issue in Visual Studio that makes the C# edit and continue for the UWP head fail with an error message if the Android, iOS or Wasm heads are loaded. Temporary unload those projects to use C# edit and continue in the Windows head.
```

Comment:
Do we have a link to the opened issue?

Reply:
We also need the dropdown on top-left of the file to be set to UWP. With a screenshot would probably be much clearer

Reply:
Even with the drop down it does not work. That trick is for XAML E&C :)